homeassistant/components/dynalite/__init__.py

```diff
@@ -251,6 +249,46 @@ async def dynalite_service(service_call: ServiceCall):
         ),
     )
 
+    def websocket_get_config_entry_data(hass, connection, msg):
+        """Get the data of a config entry."""
+        connection.send_result(
+            msg["id"],
+            {"data": dict(hass.config_entries.async_get_entry(msg["entry_id"]).data)},
```

Comment:
Isn't there already a generic api in the frontend to get and update config entry data?

I'm not a frontend expert.

Reply:
i asked and was told that there isn't since config entries may contain sensitive data. this is not the case with the dynalite config entries as the dynalite system is so backwards that it doesn't have authentication...